src/components/RoleItem/index.js

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+class RoleItem extends Component {
+  render() {
+    const {roleName, roleDateFrom, roleDateuntil, index} = this.props;
```

Comment:
```suggestion
    const { sampleRoleItem } = this.props;
```

Reply:
you could even go further in the destructuring:

```suggestion
    const { role, from, until } = this.props.sampleRoleItem;
```
and then just write those below

Reply:
@lauraferrandof thanks so much for your input! Really liked the simplicity of these props. Tried to go with the deeper destructuring option, but there is a warning regarding propTypes:
"Warning: Failed prop type: The prop `role` is marked as required in `RoleItem`, but its value is `undefined`. in RoleItem (at RoleList/index.js:13)"

I made the following amendments as suggested:
At RoleList `<RoleItem sampleRoleItem={item} key={index}/>`
At RoleItem 
```
const { role, from, until } = this.props.sampleRoleItem;
    console.log(role);
    return (
      <li className="role-list__item">
        <p className="role-list__role">{role}</p>
        <p className="role-list__date">{from} — {until}</p>
      </li>
    );
```
I checked the logs and the props are being received, not sure what I am missing here. Also I double checked our sample and it seems fine. 
I'm pushing the changes for you to review. Thanks a lot!

Reply:
@emma-martin ok, almost there! The problem is RoleItem is receiving ONLY 1 PROP, `sampleRoleItem`. After we destructure this into 3 constants, `role`, `from` and `until`, but these are not strictly props. That's why you're having trouble in your PropTypes. You just have to remove those 3 from PropTypes and leave only `sampleRoleItem: PropTypes.object.isRequired` instead :)

Or if it's less confusing, just leave the first simpler destructuring option! Whatever you feel more comfortable with, the other one was just a suggestion :)

Reply:
@lauraferrandof , awesome!!! Ready to merge, thanks so much :-D